### PR TITLE
Fixes #23420 - make dump dir locale independent

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/services/foreman_templates/template_exporter.rb [hunk 1]
```diff
@@ -77,8 +77,8 @@ def get_template_filename(template)
 
     def get_dump_dir(template)
       kind = template.respond_to?(:template_kind) ? template.template_kind.try(:name) || 'snippet' : nil
-      template_class_dir = template.model_name.human.pluralize.downcase.tr(' ', '_')
-      template_class_dir += '_templates' if template_class_dir == 'partition_tables'
+      template_class_dir = template.model_name.plural.tr(' ', '_')
```
Comment (xprazak2) [inline app/services/foreman_templates/template_exporter.rb]: I think you do not need the `.tr(' ', '_')` part as pluralized model names do not have spaces

Comment (ares) [reply app/services/foreman_templates/template_exporter.rb]: oh sure, fixed :-)

**Comment by ares** [comment]
test failures seems unrelated, smart proxy failures on 2 combinations

**Comment by xprazak2** [comment]
Merging, thanks!

**Pull request merged**